requirements-dev.txt

```diff
@@ -19,9 +19,8 @@ pandas
 pillow
 psutil
 pydantic
-pytest
-# pytest-cases
-git+https://github.com/jayqi/python-pytest-cases@packaging-version
+pytest<8
```

Comment:
pytest v8.0.0 literally just released hours ago, and there's some arcane looking error that is probably a pytest-cases incompatibility.